schemas/2015-11-01/Microsoft.Network.json

```diff
@@ -0,0 +1,222 @@
+{
+    "id": "http://schema.management.azure.com/schemas/2015-11-01/Microsoft.Network.json#",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Microsoft.Network",
+    "description": "Microsoft Network Resource Types",
+    "resourceDefinitions": {
+        "trafficManagerProfiles": {
```

Comment:
Why is the indentation here 4 instead of 2 as in the other Microsoft.Network.json file?